src/ast/passes/codegen_llvm.cpp

```diff
@@ -1161,6 +1161,19 @@ void CodegenLLVM::visit(Call &call)
     b_.CreateLifetimeEnd(buf);
     expr_ = nullptr;
   }
+  else if (call.func == "size")
+  {
+    auto &arg = *call.vargs->at(0);
```

Comment:
naming nit: maybe map_id?

Reply:
Not really, it's an argument expression which we cast to a `Map` on the following line. This is the naming used in the other map functions (e.g. `clear` and `zero`) so I followed the pattern.